docs/updating/update-to-41.md

```diff
@@ -0,0 +1,116 @@
+---
+category: update-guides
+meta-title: Update to version 41.x | CKEditor 5 Documentation
+menu-title: Update to v41.x
+order: 83
+modified_at: 2024-01-02
+---
+
+# Update to CKEditor&nbsp;5 v41.x
+
+<info-box>
+	When updating your CKEditor&nbsp;5 installations, make sure **all the packages are the same version** to avoid errors.
+
+	For custom builds, you may try removing the `package-lock.json` or `yarn.lock` files (if applicable) and reinstalling all packages before rebuilding the editor. For best results, make sure you use the most recent package versions.
+</info-box>
+
+## Update to CKEditor&nbsp;5 v41.0.0
+
+For the entire list of changes introduced in version 41.0.0, see the [release notes for CKEditor&nbsp;5 v41.0.0](https://github.com/ckeditor/ckeditor5/releases/tag/v41.0.0).
+
+Listed below are the most important changes that require your attention when upgrading to CKEditor&nbsp;5 v41.0.0.
+
+### Breaking changes to the list plugin
+
+As of the latest release, the current list plugin (often referred to as list v1) has been replaced with the {@link features/lists newer and more advanced document list plugin}, formerly known as document list (v2).
+
+The list v2 (document list) feature was implemented in 2022 to add support for block content in list items. It supported extending list markup via GHS. It did not, however, support to-do lists back then. We concentrated on bringing full list v1 functionality to this plugin. The newest release brings in the to-do list functionality and the {@link features/lists-editing#simple-lists simple list} configuration setting.
+
+We introduced the new plugin in a manner that aims to be transparent for our users, namely by physically replacing the old plugin with the new one, but retaining all namespace intact. It means, starting with release v.41.0.0 all imports of various lists-related plugins will use the new version.
+
+The old plugin has been renamed to `LegacyList` instead. The same applies to all other list-related plugins, namely: `LegacyListProperties`, and `LegacyTodoList`.
+
+Unless you need to specifically use the old v1.0 plugin in your integration, there is no need to make changes in the configuration.
```

Comment:
We should mention this here: https://github.com/ckeditor/ckeditor5/pull/15572#issuecomment-1874099428 @godai78

Reply:
Added to the Update guide.